integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -155,6 +155,15 @@
 {{- if .Values.fuse.jvmOptions }}
   {{- $fuseJavaOpts = concat $fuseJavaOpts .Values.fuse.jvmOptions }}
 {{- end }}
+
+{{- /* ===================================== */}}
+{{- /*       ALLUXIO_PROXY_JAVA_OPTS        */}}
+{{- /* ===================================== */}}
+{{- $proxyJavaOpts := list }}
+{{- $proxyJavaOpts = print "-Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}" | append $proxyJavaOpts }}
```

Comment:
Actually I don't know, does the proxy need master hostname to start? If it needs master hostname(s), then you will have to consider HA(multiple masters), like
https://github.com/Alluxio/alluxio/blob/9f2a12202e265cdc9909a3e24aa72ae6969b8726/integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml#L33

Reply:
I don't know either =D it certainly needs to know a worker to forward requests on to.

Reply:
Would you be able to try NOT give the proxy the master hostname and see if that works? My guess is no. 

If the proxy will need master hostname (or master.embedded.journal.hostnames), then I would recommend referring to the piece of code I attached and see if there's a way to switch between single/master hostname properties in your proxy code too. Please let me know if you have questions on HA deployments in K8s or in general.

Reply:
@keifgwinn when Alluxio is running in HA mode, the single master hostname as you specified will not work. please use the piece of code @jiacheliu3 referred to. thank you